docs/local_debugging.md

```
@@ -0,0 +1,50 @@
+boltz = [
+    "boltz==2.1.1",
+]
+chai = [
+    "chai_lab==0.6.1",
+    "hf_transfer==0.1.8",
+]
+```
+
+Chai requires some rdkit, that is not supported on some older CentOS, hence need to do:
+For this, we should make Docker/Apptainer images -- then one can debug in the Local environment.
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)